analyzer/java/com/engflow/bazel/invocation/analyzer/bazelprofile/BazelProfile.java

```diff
@@ -90,6 +98,10 @@ private BazelProfile(JsonObject profile) {
         .forEach(
             element -> {
               JsonObject object = element.getAsJsonObject();
+              if (!object.has(TraceEventFormatConstants.EVENT_PROCESS_ID)
+                  || !object.has(TraceEventFormatConstants.EVENT_THREAD_ID)) {
+                return;
+              }
               int pid = object.get(TraceEventFormatConstants.EVENT_PROCESS_ID).getAsInt();
```

Comment:
`getAsInt` can also throw, but I suppose that's ok?

Reply:
I've adjusted the code to use a try/catch block instead.